chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -190,6 +254,8 @@ pub struct PeerManagerActor {
     whitelist_nodes: Vec<WhitelistNode>,
     /// test-only.
     event_sink: Sink<Event>,
+
+    pub(crate) state: Arc<PeerManagerState>,
```

Comment:
Hm, torn a bit about this design: a big blog of shared state&locks feels strange in actors. OTOH, I can't name any specific reasons why I dislike this, so probably that's OK.

Reply:
discussed offline